lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -297,6 +368,15 @@ sub collect_individual_alignments {
             };
         }
 
+        my $test_name;
+        if ($ENV{GENOME_SOFTWARE_RESULT_TEST_NAME}) {
+            $test_name = $ENV{GENOME_SOFTWARE_RESULT_TEST_NAME};
+        } elsif ($self->test_name) {
```

Comment:
I actually think we always want to use the same test name as the merged result.  In what case would we want to use the environment variable?